governance/program/src/instruction.rs

```diff
@@ -110,11 +110,11 @@ pub enum GovernanceInstruction {
     ///   0. `[]` Realm account the created Governance belongs to
     ///   1. `[writable]` Account Governance account. PDA seeds: ['account-governance', realm, governed_account]
     ///   2. `[]` Account governed by this Governance
-    ///   3. `[]` Governing TokenOwnerRecord account
+    ///   3. `[]` Governing TokenOwnerRecord account (Required only if not signed by RealmAuthority)
```

Comment:
Could you remove this comment? This makes it sound like the account is optional, which it isn't since you do need to pass it in no matter.  Or you can update the language to `(Used only if not signed by RealmAuthority)`

Reply:
Done